docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 3
+sidebar_label: Get started with React
+---
+
+# React Quickstart
+
+Learn how to integrate Novu into your React.js app on the fly. Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center
+
+## Requirements
+
+To follow the steps in this quickstart, you'll need:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working React development environment.
+
+You can find the code for this project [here](https://github.com/sumitsaurabh927/react-fe-for-quickstart)
+
+## Create a new React app
+
+If you already have a working React app, please feel free to skip this section.
+
+Create a new React app by running the following command in your terminal:
+
+```bash
+npx create-react-app notifications-app
```

Comment:
`create-react-app` has been deprecated so I do not think it is a good idea to recommend its use. I think this section should be skipped for now.

Reply:
Okay. Thank you for this!
I've removed this section in this commit:
2a8b842

For future apps, what tool shall I use in place of create-react-app?

Reply:
I am not very aware what is the best React app boilerplate right now. Sorry as I can't be helpful on that part. If I discover which one will let you know.